Lombiq.Tests.UI/Extensions/ScrollingUITestContextExtensions.cs

```diff
@@ -27,8 +27,21 @@ public static void ScrollTo(this UITestContext context, By by)
     /// <summary>
     /// Scrolls to a particular set of coordinates in the document.
     /// </summary>
-    public static void ScrollTo(this UITestContext context, int x, int y) =>
+    public static void ScrollTo(this UITestContext context, int x, int y)
+    {
         context.ExecuteScript("window.scrollTo(arguments[0], arguments[1], \"instant\");", x, y);
```

Comment:
Since you are using apostrophes below as well:

```suggestion
        context.ExecuteScript("window.scrollTo(arguments[0], arguments[1], 'instant');", x, y);
```